symposion/urls.py

```diff
@@ -18,8 +18,9 @@
 urlpatterns = patterns("",
     url(r"^$", RedirectView.as_view(url="/%s/" % URL_PREFIX, permanent=True)),
     url(r"^%s/" % URL_PREFIX, include(patterns("",
-        url(r"^$", TemplateView.as_view(template_name="homepage.html"),
-            name="home"),
+#        url(r"^$", TemplateView.as_view(template_name="homepage.html"),
+#            name="home"),
+        url(r"^$", RedirectView.as_view(url="/2016/sponsors/prospectus/"),  name="home"),
```

Comment:
You'll want to specify `permanent=False` here ([reference](https://docs.djangoproject.com/en/1.8/ref/class-based-views/base/#django.views.generic.base.RedirectView.permanent)).